python/cudf_polars/cudf_polars/utils/dtypes.py

```diff
@@ -84,6 +87,71 @@ def from_polars(dtype: pl.DataType) -> plc.DataType:
         # TODO: Hopefully
         return plc.DataType(plc.TypeId.EMPTY)
     elif isinstance(dtype, pl.List):
+        # TODO: This doesn't consider the value type.
         return plc.DataType(plc.TypeId.LIST)
     else:
         raise NotImplementedError(f"{dtype=} conversion not supported")
+
+
+@cache
+def to_arrow(dtype: plc.DataType) -> pa.DataType:
```

Comment:
I think we can just replace this function with `plc.interop.to_arrow()` once https://github.com/rapidsai/cudf/pull/15257 makes it in, though I notice you cache here which is probably worth doing _there_.

Reply:
Done.